odc/stats/_cli_save_tasks.py

```diff
@@ -99,18 +107,70 @@ def save_tasks(
       - output product config
 
     """
+    setup_logging()
+
+    import logging
     from datacube import Datacube
     from .tasks import SaveTasks
     from .model import DateTimeRange
 
-    filter = {}
-    if dataset_filter:
-        filter = json.loads(dataset_filter)
-
+    _log = logging.getLogger(__name__)
     if temporal_range is not None and year is not None:
         print("Can only supply one of --year or --temporal_range", file=sys.stderr)
         sys.exit(1)
 
+    if config is None:
+        config = {}
+
+    _cfg = {k: config.get(k) for k in CONFIG_ITEMS if config.get(k) is not None}
+
+    cfg_from_cli = {
+        k: v
+        for k, v in dict(
+            grid=grid,
+            frequency=frequency,
+            gqa=gqa,
+            products=products,
+            complevel=complevel,
+            dataset_filter=dataset_filter,
```

Comment:
`overwrite` should be a part of this list.

Reply:
note to myself, `overwrite` is a bit odd atm, seems like it appends the old cache, rather than "overwrite". get on it later when I'm sure what's supposed to happen.